### PR TITLE
Fc map popups

### PR DESCRIPTION
**Pull request by Tehsurfer**

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -20,9 +20,10 @@
     <MultiFlatmapVuer ref="multi" :availableSpecies="availableSpecies" 
       @resource-selected="FlatmapSelected" :minZoom="minZoom"
       @pan-zoom-callback="panZoomcallback"
-      @ready="FlatmapReady" :featureInfo="featureInfo" :searchable="searchable" 
+      @ready="FlatmapReady" :featureInfo="featureInfo" :searchable="searchable"
+      :layerControl="layerControl"
       :initial="initial" :pathControls="pathControls" :helpMode="helpMode"
-      :displayMinimap=true :flatmapAPI="flatmapAPI"/>
+      :displayMinimap=false :flatmapAPI="flatmapAPI"/>
```
Comment (alan-wu) [inline src/App.vue:26]: Keep it on for the demo app.

File: src/App.vue [hunk 2]
```diff
@@ -73,17 +78,9 @@ export default {
...
-        "Human Female":{taxo: "NCBITaxon:9606", biologicalSex: "PATO:0000383", iconClass:"mapicon-icon_human", displayWarning:true},
-        "Human Male":{taxo: "NCBITaxon:9606", biologicalSex: "PATO:0000384", iconClass:"mapicon-icon_human", displayWarning:true},
-        "Rat":{taxo: "NCBITaxon:10114", iconClass:"mapicon-icon_rat", displayWarning:false, displayLatestChanges:true},
-        "Mouse":{taxo: "NCBITaxon:10090", iconClass:"mapicon-icon_mouse", displayWarning:true},
-        "Kember":{taxo: "ABI:1000001", displayWarning:true},
-        "Pig":{taxo: "NCBITaxon:9823", iconClass:"mapicon-icon_pig", displayWarning:true}, 
-        "Cat":{taxo: "NCBITaxon:9685", iconClass:"mapicon-icon_cat", displayWarning:true},
-        "Functional Connectivity": {id: "fc-cardio", displayWarning:true},
-        "digestive tract":{taxo: "digestive tract", displayWarning:true}
+      availableSpecies : {"Functional Connectivity":{taxo: "FunctionalConnectivity", displayWarning:true}
```
Comment (alan-wu) [inline src/App.vue:83]: Keep the previous list and replace the functional connectivity with the new one please.

**Comment by Tehsurfer** [comment]
UPDATE

the changes here are also included in the refactor #101 

assuming the structure of that branch is more favourable that is the one we will be going with

**Comment by Tehsurfer** [comment]
Closing this as it is now on the portal

**Pull request closed without merging**